### DISCUSSION TITLE
Helix in the browser, GhostText support

### DISCUSSION THREAD

**Discussion by stappersg** [helix-editor/helix#3478] [Ideas]
Hi,

[GhostText](https://ghosttext.fregante.com/) is a browser extension that connects to your editor via its own extension. This feature request is for asking an _extension_ for Helix.

In https://github.com/fregante/GhostText/blob/main/PROTOCOL.md is described how both extension talk to each other.

Maybe relevant implementations:
* [Vim](https://github.com/raghur/vim-ghost)
* [NeoVim](https://github.com/raghur/vim-ghost)

The [`demo.gif`](https://ghosttext.fregante.com/demo.gif) from the GhostText website:

![afbeelding](https://user-images.githubusercontent.com/35434094/183120857-1d24f771-9f8d-45c0-b77c-71c5bcd2d247.png)

Regards
Geert Stappers

**Comment by gglanzani** [comment]
It's there now

https://github.com/rahji/helix-ghost
